README.md

```diff
@@ -2,101 +2,14 @@
 
 The Jenkins plugin for MATLAB&reg; enables you to easily run your MATLAB and Simulink&reg; tests and generate artifacts such as *JUnit* and *TAP* test results, and *Cobertura* code or model coverage reports. You can also export your Simulink Test&#8482; Manager results. This guide shows you examples of how to configure the plugin for different objectives, considering a Microsoft&reg; Windows&reg; platform.
 
-## Configure Plugin for Freestyle Project
-To configure the plugin for a freestyle project, select **Run MATLAB Tests** from the **Add build step** list. Then, enter the value returned by the **matlabroot** function in the **MATLAB root** field.
+## Documentation 
 
-  ![new_add_build_step](https://user-images.githubusercontent.com/47204011/55624172-be54a100-57c2-11e9-9596-52d3a60ee467.png)
-  
-  ![new_enter_matlabroot](https://user-images.githubusercontent.com/51316279/70772996-28830400-1d9c-11ea-92d1-db9f054c304c.png)
-  
-### Option 1: Freestyle Project with Automatic Test Mode
-With the Jenkins plugin for MATLAB, you have the option to run your tests in either *automatic* or *custom* mode. The automatic test mode employs a default setting to run tests written using the MATLAB Unit Testing Framework and/or Simulink Test. If your source code is organized as files and folders within a project, the plugin will consider any test files in the project that have been tagged as **Test**. If your code does not leverage a project or uses a MATLAB version prior to R2019a, the plugin will consider all tests in the current Jenkins workspace including the subfolders. 
+#### Configuration Steps 
+* To learn how to configure the Jenkins Plugin for MATLAB, see step by step [configuration guide](https://github.com/mathworks/jenkins-matlab-plugin/blob/8b77953b23d8a0b0edd40dd2751219f7c3f813b8/CONFIGDOC.md) of MATLAB plugin. 
```

Comment:
Hi Nikhil,

I don't think we need step by step. Let's just say: "To learn how to configure the Jenkins Plugin for MATLAB, see [Configuration Guide]."

Reply:
Done.